elks/arch/i86/drivers/char/console-direct.c

```diff
@@ -183,41 +199,56 @@ struct tty_ops dircon_ops = {
     Console_conout
 };
 
+#if !defined(CONFIG_CONSOLE_DUAL)
 void INITPROC console_init(void)
```

Comment:
Use #ifndef rather than #if !defined. Currently for readability, we're only using #if !defined when followed by || or another check not possible with #ifndef.